docs/getting_started/bundles_starter_pack.md

```diff
@@ -0,0 +1,51 @@
+# Bundles starter pack
+
+eZ Platform application is a collection of bundles, similar to Symfony where ["everything is a bundle"](http://symfony.com/doc/current/book/bundles.html). Bundles listed below are useful additions to your clean eZ installation by using them you can improve your Platform usage.
+
+For more information about eZ bundle structure checkout:
+
+- [Bundle section](../guide/bundles.md) in our Guide.
+- List of [external bundles](https://ezplatform.com/Bundles) compatible with eZ Platform.
+
+## Everyday Platform usage
+
+|Bundle|Description|
+|------|-----------|
+|[ezmigrationbundle](https://github.com/kaliop-uk/ezmigrationbundle)|helps to handle content upgrades/migrations|
+|[EzCoreExtraBundle](https://github.com/lolautruche/EzCoreExtraBundle)|adds: configurable template variable injection; context aware Twig global variables; simplified authorization checks|
+|[TagsBundle](https://github.com/netgen/TagsBundle)|helps with taxonomy management and classification of content|
+|[NovaeZSEOBundle](https://github.com/Novactive/NovaeZSEOBundle)|simplifies SEO management|
+|[EzSystemsPrivacyCookieBundle](https://github.com/ezsystems/EzSystemsPrivacyCookieBundle)|adds privacy cookie banner into Symfony 2 application|
```

Comment:
You can remove `Symfony 2 application`, it's self-explanatory when talking about Platform. (Also below)

Reply:
Also text is outdated, it also supports symfony 3

fixed ;)